master/usage/calicoctl/configure/etcd.md

```diff
@@ -0,0 +1,124 @@
+| `etcdPassword`             | `ETCD_PASSWORD`      | Password for the given user name. Example: `password` (optional)                      | string
+| `etcdKeyFile`              | `ETCD_KEY_FILE`      | Path to the etcd key file. Example: `/etc/calico/key.pem` (optional)                  | string
+| `etcdCertFile`             | `ETCD_CERT_FILE`     | Path to the etcd client certificate, Example: `/etc/calico/cert.pem` (optional)       | string
+| `etcdCACertFile`           | `ETCD_CA_CERT_FILE`  | Path to the etcd Certificate Authority file. Example: `/etc/calico/ca.pem` (optional) | string
+
+> **Note**:
+> - If you are running with TLS enabled, ensure your endpoint addresses use HTTPS.
+> - When specifying through environment variables, the `DATASTORE_TYPE` environment
+>   is not required for etcdv3.
+> - All environment variables may also be prefixed with `CALICO_`, for example
+>   `CALICO_DATASTORE_TYPE` and `CALICO_ETCD_ENDPOINTS` etc. may also be used.
+>   This is useful if the non-prefixed names clash with existing environment
+>   variables defined on your system
+> - Previous versions of `calicoctl` supported `ETCD_SCHEME` and `ETC_AUTHORITY` environment
+>   variables as a mechanism for specifying the etcd endpoints. These variables are
+>   no longer supported. Use `ETCD_ENDPOINTS` instead.
+> - In kubeadm deployments, Calico is not configured to use the etcd run by kubeadm 
+>   on the Kubernetes master. Instead, it launches its own instance of etcd as a pod, 
+>   available at `http://10.96.232.136:6666`. Ensure you are connecting to the correct etcd 
+>   or you will not see any of the expected data.
+{: .alert .alert-info}
+
+
+
+## Examples
+
+#### Example configuration file
+
+```yaml
+apiVersion: projectcalico.org/v3
+kind: CalicoApiConfig
+metadata:
+spec:
+  etcdEndpoints: http://etcd1:2379,http://etcd2:2379,http://etcd3:2379
+  etcdKeyFile: /etc/calico/key.pem
+  etcdCertFile: /etc/calico/cert.pem
+  etcdCACertFile: /etc/calico/ca.pem
+```
+
+#### Example using environment variables
+
+```
+ETCD_ENDPOINTS=http://myhost1:2379 calicoctl get bgppeers
+```
+
+#### Example using IPv6
+
+Create a single node etcd cluster listening on IPv6 localhost `[::1]`.
+
+```
+etcd --listen-client-urls=http://[::1]:2379 --advertise-client-urls=http://[::1]:2379
+```
+
+Use the etcd IPv6 cluster:
+
+```
+ETCD_ENDPOINTS=http://[::1]:2379 calicoctl get bgppeers
+```
+
+#### Example using mixed IPv4/IPv6
+
+Create a single node etcd cluster listening on IPv4 and IPv6 localhost `[::1]`.
+
+```
+etcd --listen-client-urls=http://[::1]:2379,http://127.0.0.1:2379 --advertise-client-urls=http://[::1]:2379
+```
+
+Use the IPv6 endpoint:
+
+```
+ETCD_ENDPOINTS=http://[::1]:2379 calicoctl get bgppeers
+```
+
+Use the IPv4 endpoint:
+
+```
+ETCD_ENDPOINTS=http://127.0.0.1:2379 calicoctl get bgppeers
+```
+
+## calico/node
```

Comment:
The {{site....}} variables should be used where calico/node (section header) and calico-node, used a couple of times inside in this section.